addons/fire/functions/fnc_burnReaction.sqf

```diff
@@ -21,7 +21,7 @@ if (
     && {_unit in _unit && { !(currentWeapon _unit isEqualTo "") }}
     && {!isPlayer _unit || GVAR(dropWeapon >= 2)}
```

Comment:
it was fine but makes me very uncomfortable
```suggestion
    && {!isPlayer _unit || GVAR(dropWeapon) >= 2}
```

Reply:
that... shouldn't work?